test/integration/create-next-app/templates.test.ts

```diff
@@ -53,23 +53,13 @@ const startsWithoutError = async (
     }
   }
 }
-let testVersion
 
 describe('create-next-app templates', () => {
   if (!process.env.NEXT_TEST_CNA && process.env.NEXT_TEST_JOB) {
     it('should skip when env is not set', () => {})
     return
   }
 
-  beforeAll(async () => {
-    testVersion = (
-      await getPkgPaths({
```

Comment:
This is used to force `create-next-app` to test against the built Next.js version instead of installing latest published canary

Reply:
That's what I thought, but when I was tracing the code The only place that used it wasn't called.

But I might have forgot to build it/made different error.

Will look into it again.